<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="griffon.core.i18n.NoSuchMessageExceptionSpec" tests="2" skipped="0" failures="0" errors="0" timestamp="2016-11-14T15:58:35" hostname="testing-gce-262b1400-de9e-4b35-b7a9-cd4d4f40eb7e" time="0.004">
  <properties/>
  <testcase name="Exercise constructor with errors" classname="griffon.core.i18n.NoSuchMessageExceptionSpec" time="0.003"/>
  <testcase name="Successfully create exception" classname="griffon.core.i18n.NoSuchMessageExceptionSpec" time="0.001"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
